sources/platform/integrations/programming/webhooks/actions.md

```diff
@@ -1,37 +1,56 @@
 ---
-title: Actions
-description: Send notifications when specific events occur in your Actor (task) run or build. Dynamically add data to the notification payload when sending the notification.
+title: Webhook actions
+description: Send notifications when specific events occur in your Actor/task  run or build. Dynamically add data to the notification payload.
 sidebar_position: 2
 slug: /integrations/webhooks/actions
 ---
 
-# Actions
-
-**Send notifications when specific events occur in your Actor (task) run or build. Dynamically add data to the notification payload when sending the notification.**
+**Send notifications when specific events occur in your Actor/task  run or build. Dynamically add data to the notification payload.**
 
 ---
 
-Currently, the only available action is to send an HTTP POST request to a URL specified in the webhook. New actions will come later.
+## Send HTTP request
+
+To send notification, you can use the HTTP request action, which sends an HTTP POST request to a specified URL with a JSON payload. The payload is defined using a payload template, which is a JSON-like syntax that allows you to include variables enclosed in double curly braces `{{variable}}`. This enables the
+dynamic injection of data into the payload when the webhook is triggered.
+
+### Response management
+
+The response to the POST request must have an HTTP status code in the `2XX` range. If the response has a different status code, it is considered an error, and the request will be retried periodically with an exponential back-off:
+
+- First retry: after approximately _1 minute_
+- Second retry: after _2 minutes_
+- Third retry: after _4 minutes_
+- ...
+- Eleventh retry: after approximately _32 hours_
 
-## HTTP request
+If the request fails after _11 retries_, the system stops retrying.
 
-This action sends an HTTP POST request to the provided URL with a JSON payload. The payload is defined using a payload template, a JSON-like syntax that extends JSON with the use of variables enclosed in double curly braces `{{variable}}`. This enables the payload to be dynamically injected with data at the time when the webhook is triggered.
+### Security considerations
 
-The response to the POST request must have an HTTP status code in the `2XX` range. Otherwise, it is considered an error and the request is periodically retried with an exponential back-off: the first retry happens after roughly 1 minute, second after 2 minutes, third after 4 minutes etc. After 11 retries, which take around 32 hours, the system gives up and stops retrying the requests.
+For security reasons, include a secret token in the webhook URL to ensure that only Apify can invoke it. You can use the **Test** button in the user interface to test your endpoint.
```

Comment:
Suggestion: We can now also mention that you can use header template for this purpose.

Reply:
Will this tip work or do you think we should expand it with an example?

Reply:
Sorry, was on vacation :) I guess tip is enough.